docs/source/user-guide/vision/supported_tasks_and_formats.rst

```diff
@@ -0,0 +1,156 @@
+.. _supported_tasks_and_formats:
+
+===========================
+Supported Tasks and Formats
+===========================
+
+In order to analyze the images, labels and predictions provided in the
+:doc:`VisionData object </user-guide/vision/vision_data_class>`,
+they must be provided the accepted deepchecks format. In this section we will describe the supported formats
+for each supported task type.
+
+The VisionData object automatically validates your data formats when initialized, using the first batch of data. However,
+in addition to the automatic validations, it is also important to make sure visually that the data was provided in the
+correct format via the :func:`head <deepchecks.vision.vision_data.VisionData.head>` functionality.
+
+.. note::
+    In order to properly function, in addition for the data being in the correct format,
+    the data must also be loaded in a **shuffled** manner.
+
+We will first describe the formats which are common for all tasks, and then describe the label and
+prediction formats which vary between tasks.
+
+Common Formats
+==============
+Batch General Format
```

Comment:
Can we for these three "common formats" can we also give "examples" of "how it may look", like for the task-specific (predictions, labels...)?
I think the visuals can really help understand the meaning while reading the words

Reply:
It can also have an explanation above/after 
(e.g. "where 100 is the number of samples, x is ... and ...")